spec/factories/accounts.rb

```diff
@@ -7,7 +7,7 @@
     # Every account must have an account_user "owner" in order for the account
     # to be valid in rails. And foreign key constraints require that each
     # account_user has an account inserted before the account_user is inserted.
-    callback(:after_build, :before_create) do |account, evaluator|
```

Comment:
This callback was actually being run twice (once `after_build` and again `before_create`) resulting in duplicate `AccountUser`s.